.github/workflows/windows.yml

```diff
@@ -29,4 +29,4 @@ jobs:
         run: pip.exe install pytest pytest-xdist pyyaml
       - name: Run tests
         run: |
-          python -m pytest --numprocesses=auto -vv --maxfail=10 --ignore=tests/test_code_conventions.py --junitxml=build\test-results\pytest\test-results.xml
+          python -m pytest --numprocesses=auto -vv --maxfail=10 --ignore=tests/test_code_conventions.py --ignore=tests/test_kat_all.py --junitxml=build\test-results\pytest\test-results.xml
```

Comment:
I know we didn't test KATs before on Windows, but would that be a sensible addition while we're at this? Or rather a separate issue (and addition to the weekly runs)?

Reply:
We do run the single KAT tests on Windows; this `--ignore` flag is just to exclude the "full" KAT runs, as on all platforms. But it would probably be a good idea to expand the weekly "full" KAT testing to other platforms than Linux on x86_64. I'll open an issue for it.